pkg/controller/storagecluster/storagecluster.go

```
@@ -412,55 +412,59 @@ func (c *Controller) validateCloudStorageLabelKey(cluster *corev1.StorageCluster
 	return nil
 }
 
-func (c *Controller) runPreflightCheck(cluster *corev1.StorageCluster) error {
+func bringUpStorageCluster(cluster *corev1.StorageCluster) (bool, error) {
```

We don't need to return an error here.